code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -391,6 +391,8 @@ There are several things that need to be remembered:
 
 	apply_overlay(SHOES_LAYER)
 
+	update_mutant_bodyparts()
```

Comment:
this does nothing. legs arent mutant bodyparts anymore.

Reply:
It does work, but that might just be because update_mutant_bodyparts() itself calls update_body_parts(). Would it be better to call the latter directly here, instead?

Reply:
you are correct. call it directly.

Reply:
Okay do it then

Reply:
```suggestion
	update_body_parts()
```

Reply:
I will assume it works